harness/determined/common/experimental/experiment.py

```diff
@@ -52,17 +52,31 @@ def __init__(
         self._id = experiment_id
         self._session = session
 
+        self.config = None  # type: Optional[Dict[str, Any]]
+
     @property
     def id(self) -> int:
         return self._id
 
-    def _get(self) -> bindings.v1Experiment:
+    def _get(self, session: Optional[api.Session] = None) -> bindings.v1Experiment:
```

Comment:
Proposal not because I think it's necessarily the right choice but because I want to hear what you think. remove `_get` from this module completely and rely exclusively on `reload`. Once the object is reloaded, then access the updated attributes.

Per an earlier comment of yours, I think you're intending to be aggressive about hydrating whenever it's free. This makes that design pretty explicit.

Reply:
i think that's a good idea. i originally only separated it because i had thought we might want to support configurable sessions per `_get`. but per the other comments, i think i'm just going to follow ilia's suggestion of default retries. so i can and will get rid of this.